### PR TITLE
Makes whispering hide your voice. And deaf people now can't see your voice

### PR DESCRIPTION
**Pull request by farie82**
## What Does This PR Do
Makes whispering hide your voice and makes deaf people unable to "hear"/see your voice

## Why It's Good For The Game
It will make kidnapping more interesting since the kidnapper can actually talk to the victim without giving their identity away

## Images of changes
image

It will not affect whispering on the comms
image

## Changelog
:cl:
tweak: Whispering now hides your voice
tweak: Deaf people now can't see your voice when you talk
/:cl:

### CODE REVIEW

**Comment by Cazdon** [comment]
In general this looks like it might be a pretty sizable buff to antags being able to hide their voice via whispering. However, it's a buff in the right spot I believe. Pulling off deceptions and hiding your identity are fairly roleplay oriented, and making roleplay related options stronger is a plus in my book.

**Comment by farie82** [comment]
> In general this looks like it might be a pretty sizable buff to antags being able to hide their voice via whispering. However, it's a buff in the right spot I believe. Pulling off deceptions and hiding your identity are fairly roleplay oriented, and making roleplay related options stronger is a plus in my book.

I was debating making it a balance change in the CL or just a tweak. People can already achieve this by using me "whispers "Heyo boiiiiiii
That will probably be against the rules but it's possible.

If the maints label it as balance then I'll change it

File: code/modules/mob/hear_say.dm [hunk 1]
```diff
@@ -99,9 +99,9 @@
 		if(speaker == src)
 			to_chat(src, "<span class='warning'>You cannot hear yourself speak!</span>")
 		else
-			to_chat(src, "<span class='name'>[speaker_name]</span>[speaker.GetAltName()] talks but you cannot hear [speaker.p_them()].")
+			to_chat(src, "<span class='name'>[speaker_name]</span> talks but you cannot hear [speaker.p_them()].")
```
Comment (farie82) [inline code/modules/mob/hear_say.dm]: Todo for self. Change speaker_name to speaker.name

**Comment by farie82** [comment]
Closing and reopenening to give travis a reason to live again

**Comment by AffectedArc07** [comment]
Can we get a better screenshot of what this does and doesnt do?
```dm
/obj/item/card/id/captains_spare
	name = "captain's spare ID"
	desc = "The spare ID of the captain."
	icon_state = "gold"
	item_state = "gold_id"
	registered_name = "Captain"
	assignment = "Captain"
```

The captain's spare ID forces your name to be captain if you have your identity concealed, so its misleading of what it shows (Their ID assigned name, their real name, their job?)

**Comment by farie82** [comment]
> Can we get a better screenshot of what this does and doesnt do?
> 
> ```dm
> /obj/item/card/id/captains_spare
> 	name = "captain's spare ID"
> 	desc = "The spare ID of the captain."
> 	icon_state = "gold"
> 	item_state = "gold_id"
> 	registered_name = "Captain"
> 	assignment = "Captain"
> ```
> 
> The captain's spare ID forces your name to be captain if you have your identity concealed, so its misleading of what it shows (Their ID assigned name, their real name, their job?)

It takes the name of the ID if any:
image

**Pull request merged**